arbeitszeit_web/www/presenters/list_registered_hours_worked_presenter.py

```diff
@@ -0,0 +1,38 @@
+from dataclasses import dataclass
+
+from arbeitszeit.use_cases import list_registered_hours_worked
+from arbeitszeit_web.formatters.datetime_formatter import DatetimeFormatter
+
+
+@dataclass
+class RegisteredHoursWorked:
+    hours: str
+    worker_name: str
+    worker_id: str
+    registered_on: str
+
+
+@dataclass
+class ViewModel:
+    registered_hours_worked: list[RegisteredHoursWorked]
+
+
+@dataclass
+class ListRegisteredHoursWorkedPresenter:
+    datetime_formatter: DatetimeFormatter
+
+    def present(self, response: list_registered_hours_worked.Response) -> ViewModel:
+        registered_hours_worked = [
+            RegisteredHoursWorked(
+                hours=str(round(record.hours, 2)),
+                worker_name=record.worker_name,
+                worker_id=str(record.worker_id),
+                registered_on=self.datetime_formatter.format_datetime(
+                    date=record.registered_on,
+                    zone="Europe/Berlin",
```

Comment:
In another PR, this could be defined via an environment variable. Even better practice is that users should be able to set their specific timezone in their account settings by themselves.

Reply:
Yeah, I agree that this should be configurable..